src/diffTool/rulesProcessor.test.ts

```diff
@@ -4,43 +4,30 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import { NodeDiff } from "../../src/diffTool/jsonDiff";
-import { applyRules } from "../../src/diffTool/rulesProcessor";
+import { NodeDiff } from "./jsonDiff";
+import { applyRules } from "./rulesProcessor";
 import * as chai from "chai";
 import fs from "fs-extra";
 import tmp from "tmp";
-import sinon from "sinon";
-import { ramlToolLogger } from "../../src";
 import chaiAsPromised from "chai-as-promised";
 
 /* eslint-disable @typescript-eslint/no-use-before-define*/
 
 const expect = chai.expect;
 chai.use(chaiAsPromised);
 
-let loggerSpy;
```

Comment:
what is the issue here? Is it related to the log level

Reply:
When the last test in the logger test file (which switches the loglevel to debug) is included in the run, it causes the spy in this test to fail. I tried a bunch of things to get it to work, like resetting the level in the logger test and explicitly setting the level here before creating the spy, but nothing worked.